### PR TITLE
remove use of tc-base

### PR DESCRIPTION
**Pull request by djmitche**
This really only affects this module's tests

tc-lib-api, crazily, require's some modules from tc-lib-testing's tests, but I'll remove those.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,14 +35,14 @@
...
+    "babel-preset-taskcluster": "^3.0.0",
     "eslint-config-taskcluster": "^2.0.0",
     "eslint-plugin-taskcluster": "^1.0.0",
     "mocha": "2.0.1",
-    "taskcluster-lib-config": "^0.8.8",
-    "taskcluster-base": "^0.12.1",
-    "source-map-support": "^0.4.0"
+    "source-map-support": "^0.4.0",
+    "taskcluster-lib-api": "^1.3.1",
+    "taskcluster-lib-config": "^0.8.8"
```
Comment (jonasfj) [inline package.json:45]: We should away from this... but it doesn't have to be in this PR...

**Comment by jonasfj** [comment]
# `r+`

Assuming the tests pass... Note: you're welcome to remove the old config thing and fix travis... Otherwise I just trust that you ran tests locally.

File: package.json (hunk 1 above)
Comment (djmitche) [reply package.json:45]: Yeah, that one didn't change here, just moved around in the JSON.

**Comment by djmitche** [comment]
```
  fakeauth
Constants file does not exist, setting constants to {}
{ hasTestScope: true }
{ hasTestScope: true }
    ✓ using a rawClientId (83ms)
{ hasTestScope: true }
{ hasTestScope: true }
    ✓ using authorizedScopes 
{ hasTestScope: true }
{ hasTestScope: true }
    ✓ using temp creds 

  testing.schema
Constants file does not exist, setting constants to {}
    ✓ case1.json 
    ✓ case2.json 

  testing.schema w. schemaPrefix
Constants file does not exist, setting constants to {}
    ✓ case1.json 
    ✓ case2.json 

  testing (utilities)
    ✓ sleep 
    ✓ poll (success) 
    ✓ poll (too-few iterations) 
    ✓ poll (terminate early) 
```

:)

**Comment by djmitche** [comment]
published as 1.0.3

**Pull request merged**